### ISSUE TITLE
Improve site appearance on mobile

### ISSUE THREAD

**Issue by bitspittle** [varabyte/kobweb-site#4]
We should use responsive styles that feel good on a mobile device instead of it looking like a super tiny desktop mode

See also this reddit comment: https://www.reddit.com/r/Kotlin/comments/r94xg3/comment/hnascvf/?utm_source=share&utm_medium=web2x&context=3

**Comment by bitspittle** [comment]
Oh nice I didn't even check.

Have you seen how to do responsive styles? They're mentioned in the README
but I haven't documented them well yet.

Something like:

```
ComponentStyle("blah") {
   base { ... applies to mobile ... }
   Breakpoint.MD { ... applies to desktop ... }
}
```

You can also use media queries in the site's StyleSheet

**Comment by deepanshu11madan** [comment]
Nice, yeah I saw how the grid items were done. 

But most optimizations I made were removing hardcoded values for spacing and width. Like earlier, there would be a lot of margin space on the sides of the sections and the fixed height & width of grid items didn't look good on phones.

So now it looks like this:
<img width="259" alt="Screen Shot 2021-12-08 at 8 29 33 AM" src="https://user-images.githubusercontent.com/47491495/145246381-b825e83f-cdc0-4a45-b5e9-243cca9d431b.png">
:
